testing/runtests.py

```diff
@@ -697,6 +698,18 @@ def main():
         tests = sorted(glob.glob('[0-9][0-9][0-9]_*'))
     else:
         tests = list(itertools.chain.from_iterable(tests))
+
+    if args.exclude_ids:  # test ids are given by user
+        for id in list(itertools.chain.from_iterable(args.exclude_ids)):
+            x=glob.glob('%s_*'%id)
```

Comment:
I think it is better to do `x=glob.glob('{:03d}_*'.format(id))` here instead of 3 different tests, or doesn't this work?

Reply:
I wanted it to be consistent with the other code that was already present. the line like:
```
print(glob.glob('%03d_*'%int(id))`
```
works in Python 2 and 3 so would indeed make the code a bit smaller.
I'll update the PR.

Reply:
Updated.